lib/api-call.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const axios = require('axios');
+
+module.exports = {
+  getResponses: function(baseUrl, surveyID, endPoint, apiToken, apiTokenSecret) {
+    axios.get(baseUrl + '/' + surveyID + '/' + endPoint + '?api_token=' + apiToken + '&api_token_secret=' + apiTokenSecret)
```

Comment:
you could do with something to build the GET url query params up based on an object.
https://nodejs.org/api/querystring.html
`querystring.stringify({ foo: 'bar', baz: ['qux', 'quux'], corge: '' })`

for the building the `baseUrl + '/' + surveyID + '/' + endPoint` to check you're doing: 
```
`${baseUrl}/${surveyID}/${endPoint}`
```
yeh?